.github/CONTRIBUTING.md

```diff
@@ -1,34 +0,0 @@
-# Contributing to Lean
```

Comment:
I didn't actually know we (still) had this file... I copied a few bullet points to the new one